enhancements/olm/olm-managed-operator-metrics.md

```diff
@@ -78,21 +77,18 @@ bundle
         └── annotations.yaml
 ```
 
-OLM would then need to be updated to identify the `ServiceMonitors`
-and `PrometheusRules` resources and deploy them as part of an
-`InstallPlan`. There is no way for OLM to identify which Prometheus
-Operator ServiceAccount should be granted the appropriate RBAC
-privileges to view the `ServiceMonitor` and `PrometheusRule` resource
-events in the namespace that the operator is being deployed to. As
-such, the cluster admin will need to configure the Prometheus Operator
-to watch for events in the correct namespace.
+OLM will then install the `ServiceMonitors` and `PrometheusRules` resources as part of an `InstallPlan`.
+
+> Note: If a bundle includes a `ServiceMonitor` or `PrometheusRule` resource the CSV should add a dependency to each CRD so OLM can install the prometheus operator on vanilla kubernetes clusters.
+
+On vanilla Kubernetes clusters there is no way for OLM to identify which Prometheus Operator ServiceAccount should be granted the appropriate RBAC privileges to view the `ServiceMonitor` and `PrometheusRule` resource events in the namespace that the operator is deployed in.
+As such, the cluster admin will need to configure the Prometheus Operator to watch for events in the correct namespace.
 
 ### OpenShift Monitoring Prometheus Operator Support
 
 #### Requirements
 
-Once OLM offers bundle support for the `ServiceMonitor` and `PrometheusRule` resources, a sanctioned set of OLM managed operators should be able to report metrics to the OpenShift Monitoring Prometheus Operator.
-
+Once OLM offers bundle support for the `ServiceMonitor` and `PrometheusRule` resources, a sanctioned set of OLM managed operators should be able to expose metrics that are collected by the OpenShift Monitoring Prometheus Instance.
```

Comment:
How would one become a sanctioned OLM operator? Is openshift going to block community operators that have these in any way? We have a community operator that is in a weird gray zone so just wondering what it's status would look like.

Reply:
There is nothing that stops community operators from exposing metrics to the OpenShift Monitoring Prometheus Instance provided that they follow the steps outlined in this document. If the operator wants to have these metrics available in Telemeter they'd need to work with the monitoring team.